src/forms/checkbox-group.jsx

```diff
@@ -103,13 +103,13 @@ CheckboxGroup.propTypes = {
   name: PropTypes.string.isRequired,
   options: Types.options.isRequired,
   initialHideReveals: PropTypes.bool,
-  label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
+  label: PropTypes.node.isRequired,
   onChange: PropTypes.func,
   id: PropTypes.string,
   inline: PropTypes.bool,
   value: PropTypes.oneOfType([Types.value, PropTypes.arrayOf(Types.value)]),
-  hint: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
-  error: PropTypes.oneOfType([PropTypes.string, PropTypes.node])
+  hint: PropTypes.node,
+  error: PropTypes.node
```

Comment:
Turns out `PropTypes.node` is not what I thought it was. I got most of the way through a comment about not allowing strings and then went to look at the docs, and actually this is fine.